common/djangoapps/third_party_auth/models.py

```diff
@@ -161,7 +161,11 @@ def clean(self):
     def provider_id(self):
         """ Unique string key identifying this provider. Must be URL and css class friendly. """
         assert self.prefix is not None
-        return "-".join((self.prefix, ) + tuple(getattr(self, field) for field in self.KEY_FIELDS))
+        fields = []
```

Comment:
@bdero nit: might make sense to initialize `fields` list with `self.prefix`, i.e.

    fields = [self.prefix]
    ....
    return "-".join(fields)

On the other hand you might want to keep the semantics of `fields` list keeping actual field values. So totally up to you :)